table/tables/gen_expr.go

```diff
@@ -78,5 +80,9 @@ func simpleResolveName(node ast.ExprNode, tblInfo *model.TableInfo) (ast.ExprNod
 	if _, ok := node.Accept(&nr); !ok {
 		return nil, errors.Trace(nr.err)
 	}
+	sc := variable.MostRestrictStateContext()
```

Comment:
The empty **sc** has `IgnoreTruncate`, `OverflowAsWarning` and `TruncateAsWarning` as `false`,  just equals to `MostRestrictStateContext` returns.